components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -130,6 +128,8 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <paramfile nu_com="ECA" use_cn=".true." use_crop=".false." >lnd/clm2/paramdata/clm_params.cbgc.c07292018.nc</paramfile>
 <paramfile use_crop=".false." use_cn=".true."  >lnd/clm2/paramdata/clm_params_c211124.nc</paramfile>
 <paramfile use_crop=".false." use_fates=".true."  >lnd/clm2/paramdata/clm_params_c211124.nc</paramfile>
+<paramfile nu_com="RD" use_crop=".false." >lnd/clm2/paramdata/clm_params_c180524_phs.nc</paramfile>
```

Comment:
Since crops use RD it's not clear why a different parameter file is used when crops are off, unless this file has fewer PFTs. But, the parameter file with crops should also reflect any changes to parameters for natural vegetation that are in this parameter file.

Reply:
@bbye The parameter file in this PR (`clm_params_c180524_phs.nc`) is for 25 pfts. The new crop parameter file (`clm_params_c230517_phs_50pfts.nc`) was made by combining the parameter file in this PR (`clm_params_c180524_phs.nc`) and an earlier crop parameter file (`clm_params_c230517_phs_50pfts.nc`). 

So ideally, using `clm_params_c230517_phs_50pfts.nc` file in this PR should work but it has not been tested.

Reply:
Can there be one params file for all cases?